react-router-redux/react-router-redux.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for react-router-redux v2.0.4
+// Project: https://github.com/rackt/react-router-redux
+// Definitions by: Stepan Mikhaylyuk <https://github.com/stepancar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../react-router/react-router.d.ts"/>
+///<reference path="../redux/redux.d.ts"/>
+
+declare module ReactRouterRedux {
+    interface transitionGoFunc {
+        (n?: number): { type: string; payload: Object }
+    }
+    interface transitionToLocationFunc {
+        (nextLocation: string): { type: string; payload: Object }
+    }
+    interface transitionFunc {
+        (): { type: string; payload: Object }
+    }
+    interface RouteActions {
+        push: transitionToLocationFunc;
+        replace: transitionToLocationFunc;
+        go: transitionGoFunc;
+        goBack: transitionFunc;
+        goForward: transitionFunc;
+    }
+    interface LocationState {
+        location: string;
+    }
+    interface Middleware extends Redux.Middleware {
+        listenForReplays: (store: Redux.Store) => any;
+    }
+}
+
+declare module "react-router-redux" {
+    import History = ReactRouter.History;
```

Comment:
This should be `History` from "../react-router/history.d.ts". Otherwise, if you construct a history via `createHistory`, there's no way to get it to typecheck.